### PR TITLE
Add support button

### LINKED ISSUE #277: add an add to the physical book on Amazon
I was thinking that maybe having an add (beside the links in the first page) to the book in the website will be nice to have.

### PR DESCRIPTION
**Pull request by rikhuijzer**
- Closes #277 

## Desktop Preview

desktop preview

## Mobile Preview

image

Let's merge when both reviewers are happy

### CODE REVIEW

**Comment by lazarusA** [comment]
is too hidden at the bottom. Could we might have it at the top? and use a [fa-fa coffee] ? cup in green? or other colour, but not without it.

**Comment by lazarusA** [comment]
I see. My only issue is that is not visible enough. Maybe putting a box around it [in green, red, orange or], like in the one you mentioned in the issue?, I could take a look at later on how to include it in your span element.

**Review by storopoli — APPROVED** [review]
Once @lazarusA is fine with the placement and details we can merge.

File: metadata.yml [hunk 1]
```diff
@@ -5,7 +5,16 @@ author:
   - Jose Storopoli
   - Rik Huijzer
   - Lazaro Alonso
-html-license: '<a href="http://creativecommons.org/licenses/by-nc-sa/4.0/">CC BY-NC-SA 4.0</a>'
+html-license: >-
+  <span style="display: flex; justify-content: center;">
```
Comment (lazarusA) [inline metadata.yml:9]:
```suggestion
 <span style="width: 200px; height: 35px; display: flex; justify-content: center; border: 1.5px solid rgb(67, 72, 54);border-radius: 10px;">
```
we could go for red, green or completely black, now is kinda dark and probably is already ok. Any colour will do, feel free to merge @rikhuijzer, with the suggestion or without it.

**Review by lazarusA — APPROVED** [review]
please, take a look at the suggestion.

**Pull request merged**